.github/workflows/prepare-release-branch.yml

```diff
@@ -0,0 +1,115 @@
+name: Prepare release branch
+on:
+  workflow_dispatch:
+
+jobs:
+  prereqs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Verify prerequisites
+        run: |
+          if [[ $GITHUB_REF_NAME != main ]]; then
+            echo this workflow should only be run against main
+            exit 1
+          fi
+
+          if ! grep --quiet "^## Unreleased$" CHANGELOG.md; then
+            echo the change log is missing an \"Unreleased\" section
+            exit 1
+          fi
+
+  create-pull-request-against-release-branch:
+    runs-on: ubuntu-latest
+    needs:
+      - prereqs
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Create release branch
+        run: |
+          version=$(.github/scripts/get-version.sh)
+          if [[ $version =~ ^([0-9]+)\.([0-9]+)\.0$ ]]; then
+            release_branch_name="release/v${version}"
+            git ls-remote --exit-code --heads origin refs/heads/$release_branch_name
+            if [ "$?" == "0" ] ; then 
+              echo "release branch $release_branch_name already exists"
+              exit 1 
+            fi
```

Comment:
In otel java, release_branch_name is set to `release_branch_name=release_branch_name=$(echo $version | sed -E 's/([0-9]+)\.([0-9]+)\.0/release\/v\1.\2.x/')`, which is more complicated than needed for this repo.

In this repo, we confirm the release branch doesn't already exist, which is needed to prevent duplicate releases since the version of `open-telemetry/semantic-convention` is manually updated:
```            
git ls-remote --exit-code --heads origin refs/heads/$release_branch_name
if [ "$?" == "0" ] ; then 
  echo "release branch $release_branch_name already exists"
  exit 1 
fi
```